src/commands/admin_commands.py

```
@@ -403,3 +403,85 @@ async def package_update_query(interaction: discord.Interaction):
+
+        # Notify the user and send new assignment selector
+        try:
+            # Send assignment selector DM
+            from ..handlers.assignment_handler import send_assignment_dm
+
+            await send_assignment_dm(bot, db, user)
+
+            await interaction.channel.send(
+                f"✅ Sent new assignment selector to {user.mention}", delete_after=10
+            )
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)